src/topp/FileConverter.cpp

```diff
@@ -351,6 +356,14 @@ class TOPPFileConverter :
     registerDoubleOption_("lossy_mass_accuracy", "<error>", -1.0, "Desired (absolute) m/z accuracy for lossy compression (e.g. use 0.0001 for a mass accuracy of 0.2 ppm at 500 m/z, default uses -1.0 for maximal accuracy).", false, true);
 
     registerFlag_("process_lowmemory", "Whether to process the file on the fly without loading the whole file into memory first (only for conversions of mzXML/mzML to mzML).\nNote: this flag will prevent conversion from spectra to chromatograms.", true);
+#ifdef OPENMS_WINDOWSPLATFORM
+    String net_executable = "";
+#else
+    String net_executable = "mono";
+#endif
+    registerInputFile_("NET_executable", "<executable>", net_executable, "The .NET framework executable. Only required on linux and mac.", false, true, ListUtils::create<String>("skipexists"));
```

Comment:
Not sure this is a good idea. Workflow defaults will break across platforms.
How about: by default the param value is emtpy which gets interpreted as Mono on non-win systems internally.